policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/living_tax_credit/max_amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Northwest Territories allows for this maximum amount under the cost of living tax credit.
+values:
+  2022-12-31: 942 
+metadata:
+  unit: currency-CAD
+  label: Northwest Territories cost of living tax credit max amount
+  reference:
+  - title: Government of Canada - Northwest Territories Cost of Living Credit
+    href: https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/northwest-territories/5012-tc.html 
+  - title: Government of Canada - Income Tax Act
```

Comment:
could you add the section of the law f.e. section 3.6 (a) or something like that

Reply:
in all yaml files